ledger/sandbox-classic/src/main/scala/platform/sandbox/stores/ledger/sql/SqlLedger.scala

```diff
@@ -211,8 +213,13 @@ private[sandbox] object SqlLedger {
 
         ledgerDao
           .storePackageEntry(newLedgerEnd, packages, None)
-          .transform(_ => (), e => sys.error("Failed to copy initial packages: " + e.getMessage))(
-            DEC)
+          .flatMap { _ =>
+            ledgerDao.updateLedgerEnd(newLedgerEnd)
+          }(DEC)
```

Comment:
You probably want to expand this one too.

```suggestion
          }(DirectExecutionContext)
```

Reply:
Agreed, but I was too lazy and we probably won't merge this PR anyway.